dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -476,30 +485,91 @@
             </xsl:for-each>
```

Comment:
Although not a part of this patch's changes, there's an issue with this for-each. It iterates over the dc:identifier:* elements, but this results in only the first one of each being processing based on how the relatedIdentifierTemplate is called. Modifications are needed (somewhere) so that iteration is over all
`./doc:element/doc:element/doc:field[@name='value']/text()`
values. This would handle multiple dc:identifier:* values including language variants and multiple possible values for each one.